File: Aula 04 - Estruturas de Decisao/java_logic/desvio.multiplo.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Digite a quantidade em estoque: ");
        int quantidade = scanner.nextInt();
    
        if (quantidade < 1) {
            System.out.println("Status: em falta");
        } else if (quantidade < 5) {
            System.out.println("Status: alerta de estoque baixo.");
        } else {
            System.out.println("Status: estoque OK.");
        }
        
        scanner.close();
    }
}

File: Aula 04 - Estruturas de Decisao/cpp_logic/desvio_multiplo.cpp
#include <iostream>
using namespace std;

int main() {
    int quantidade;
    cout << "Digite a quantidade em estoque: ";
    cin >> quantidade;

    if (quantidade < 1) {
        cout << "Status: em falta." << endl;
    } else if (quantidade < 5) {
        cout << "Status: alerta de estoque baixo." << endl;
    } else {
        cout << "Status: estoque OK." << endl;
    } 

    return 0;
}
